Tests/Data/ThermoHydroMechanics/Linear/Point_injection/square_1e2.prj

```diff
@@ -0,0 +1,288 @@
+            <prefix>square_1e0</prefix>
+            <timesteps>
+                <pair><repeat>1</repeat>
+                <each_steps>10</each_steps></pair>
+            </timesteps>
+                <variables>
+                    <variable>displacement</variable>
+                    <variable>pressure</variable>
+                    <variable>temperature</variable>
+                    <variable>sigma</variable>
+                    <variable>epsilon</variable>
+                </variables>
+        </output>
+    </time_loop>
+    <parameters>
+        <parameter>
+            <name>E</name>
+            <type>Constant</type>
+            <value>5000000000</value>
+        </parameter>
+        <parameter>
+            <name>nu</name>
+            <type>Constant</type>
+            <value>0.3</value>
+        </parameter>
+        <parameter>
+            <name>k</name>
+            <type>Constant</type>
+            <value>2e-20</value>
+        </parameter>
+        <parameter>
+            <name>S</name>
+            <type>Constant</type>
+            <value>0</value>
+        </parameter>
+        <parameter>
+            <name>alpha</name>
+            <type>Constant</type>
+            <value>1</value>
+        </parameter>
+        <parameter>
+            <name>phi</name>
+            <type>Constant</type>
+            <value>0.16</value>
+        </parameter>
+
+        <parameter>
+            <name>rho_sr</name>
+            <type>Constant</type>
+            <value>2290</value>
+        </parameter>
+        <parameter>
+            <name>alpha_s</name>
+            <type>Constant</type>
+            <value>1.5e-5</value>
+        </parameter>
+        <parameter>
+            <name>beta_f</name>
+            <type>Constant</type>
+            <value>4e-4</value>
+        </parameter>
+        <parameter>
+            <name>lambda_s</name>
+            <type>Constant</type>
+            <value>1.838</value>
+        </parameter>
+        <parameter>
+            <name>C_s</name>
+            <type>Constant</type>
+            <value>917.654</value>
+        </parameter>
+        <parameter>
+            <name>T0</name>
+            <type>Constant</type>
+            <value>273.15</value>
+        </parameter>
+        <parameter>
+            <name>displacement0</name>
+            <type>Constant</type>
+            <values>0 0</values>
+        </parameter>
+        <parameter>
+            <name>pressure_ic</name>
+            <type>Constant</type>
+            <values>0</values>
+        </parameter>
+        <parameter>
+            <name>dirichlet0</name>
+            <type>Constant</type>
+            <value>0</value>
+        </parameter>
+        <parameter>
+            <name>Neumann0</name>
+            <type>Constant</type>
+            <value>0</value>
+        </parameter>
+        <parameter>
+            <name>temperature_ic</name>
+            <type>Constant</type>
+            <value>273.15</value>
```

Comment:
This value is different from the value in the documentation, p.7.

Reply:
The parameters written in the documentation is made long time ago, there are some updates for the current tests. I will check and update all the parameters in the pdf.

Reply:
The documentation is updated.